src/bin/cli.js

```diff
@@ -176,5 +176,10 @@ const verifyInputGetData =  async (rd, asyncApiFilepath,scenarioFile,basedir) =>
   const manager = RequestManager();
   await manager.createReqHandler(structuredData);
   await manager.startOperations();
-}());
+};
+
+function Main () {
+  cli();
+}
 
+Main();
```

Comment:
Why the wrapper function? Is it not easier just to do
```suggestion
cli();
```